src/modal-manager/themes/orion-fusion-theme/orion-fusion-theme.scss

```diff
@@ -0,0 +1,7 @@
+:global {
```

Comment:
Should these be applied as a global theme? Does this affect other themes by adding these as global? I would have thought these should be aggregated by the theme provider?

I'm looking at the port theme PR (https://github.com/cerner/terra-core/pull/2702) have will have the same questions there 😄

Reply:
I was seeking consistency with that PRs implementation. I'll defer to decisions made there.

Reply:
We default to global in our webpack config. It doesn't hurt to be explicit. https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js#L65

Yes, these files are aggregated to root or scoped files, which are then included as an entry point:
```
:root {
  @import ...modal-manager/themes...orion-fusion-theme.scss
}
``` 
```
.orion-fusion-theme {
  @import ...modal-manager/themes...orion-fusion-theme.scss
}
``` 
Scoping these globally avoids a second scope layer. This should not be an issue because theme files are not imported in the component. Someone could manually import these, but that's on them.